release/models/policy/openconfig-policy-types.yang

```diff
@@ -165,6 +172,7 @@ module openconfig-policy-types {
     type union {
       type uint32;
       type yang:hex-string;
+      type oc-yang:hex-string-prefixed;
```

Comment:
In the case above, there is a replacement of the type whereas here is an addition preserving the previous.  Are we removing any ambiguity by keeping the union of the old + new here?

Reply:
updated